index.html

```diff
@@ -0,0 +1,123 @@
+  <meta property="og:title" content=" " />
+  <meta property="og:description" content=" " />
+  <meta property="og:url" content="https://www.codeit.kr" />
+  <meta property="og:type" content="website" />
+  <meta name="twitter:card" content="summary_large_image" />
+  <meta name="twitter:image" content="./resources/images/img_home_bottom.png" />
+  <meta name="twitter:title" content=" " />
+  <meta name="twitter:description" content="" />
+  <link rel="shortcut icon" href="./resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="icon" href="./resources/images/common/favicon.ico" type="image/x-icon" />
+  <link rel="preload" href="./resources/css/main.css" as="style" onload="this.onload=null;this.rel='stylesheet'" />
+</head>
+<body>
+  <a href="#container" class="skipNav">본문 바로가기</a>
+
+  <header>
+    <nav class="header__top">
+      <h1 class="header__logo">
+        <a href="/" title="판다마켓 홈">
+          <img src="./resources/images/common/logo_pandaMarket.png" alt="판다마켓 홈" />
+        </a>
+      </h1>
+      <div class="header__login">
+        <a href="./html/login.html" class="btn_login">로그인</a>
+      </div>
+    </nav>   
+  </header>  
+  
+  <main class="content" id="container">
+    <div class="visual">
+      <div class="inner">
+        <div class="visual__content">
+          <h2>일상의 모든 물건을<br />거래해 보세요</h2>
+          <a href="./html/items.html" class="btn_shopping"><span>구경하러 가기</span></a>
+        </div>     
+      </div>
+    </div> 
+
+    <div class="inner">
+      <section class="shopGuide">
+        <div class="shopGuide__box">
+          <div class="shopGuide__box--img">
+            <img src="./resources/images/img_home_01.png" alt="">
+          </div>
+          
+          <div class="shopGuide__box--info">
+            <em>Hot item</em>
+            <h2>인기 상품을<br />확인해 보세요</h2>            
+            <span>가장 HOT한 중고거래 물품을<br />판다마켓에서 확인해 보세요</span>
+          </div>
+        </div>
+
+        <div class="shopGuide__box">
+          <div class="shopGuide__box--img">
+            <img src="./resources/images/img_home_02.png" alt="">
+          </div>
+          
+          <div class="shopGuide__box--info">
+            <em>Search</em>
+            <h2>구매를 원하는 <br /> 상품을 검색하세요</h2>            
+            <span>구매하고 싶은 물품은 검색해서 <br /> 쉽게 찾아보세요</span>
+          </div>
+        </div>
+
+        <div class="shopGuide__box">
+          <div class="shopGuide__box--img">
+            <img src="./resources/images/img_home_03.png" alt="">
+          </div>
+          
+          <div class="shopGuide__box--info">
+            <em>Register</em>
+            <h2>판매를 원하는 <br /> 상품을 등록하세요</h2>            
+            <span>어떤 물건이든 판매하고 싶은 상품을 <br /> 쉽게 등록하세요</span>
+          </div>
+        </div>
+        
+      </section>
+    </div>
+  </main>
+ 
+  <article class="banner">
+    <div class="inner">
+      <div class="banner__content">
+        <h2> 믿을 수 있는 <br> 판다마켓 중고거래</h2>      
```

Comment:
❗️ 수정요청
공백이 들어가 있으면 의도치 않게 동작할 수 있습니다.
필요한 경우만 공백을 넣으시는 것을 추천드려요~

```suggestion
        <h2>믿을 수 있는 <br>판다마켓 중고거래</h2>      
```